locations/commands/insights.py

```diff
@@ -224,6 +224,12 @@ def lookup_code(wikidata_code):
             wikidata_dict[wikidata_code] = record
             return record
 
+        def get_brand_name(item_tags: dict):
+            # Prefer English brand name for insights application (https://www.alltheplaces.xyz/wikidata.html).
+            if brand_name := item_tags.get("brand:en"):
+                return brand_name
+            return item_tags.get("brand")
```

Comment:
Python lets us shorten this up to this. What do you think?

```suggestion
            return item_tags.get("brand:en") or item_tags.get("brand")
```

Reply:
For sure better and done!